src/zenml/zen_server/routers/runs_endpoints.py

```diff
@@ -107,13 +113,24 @@ def get_run(
         run_id: ID of the pipeline run to get.
         hydrate: Flag deciding whether to hydrate the output model(s)
             by including metadata fields in the response.
+        refresh_status: Flag deciding whether we should try to refresh
+            the status of the pipeline run using its orchestrator.
 
     Returns:
         The pipeline run.
     """
-    return verify_permissions_and_get_entity(
+    run = verify_permissions_and_get_entity(
```

Comment:
Same with the permissions for the orchestrator

Reply:
Again, the comment above. Resolving this for now.